service/src/main/java/bio/terra/pipelines/stairway/imputation/SubmitCromwellRunSetStep.java

```diff
@@ -47,8 +47,7 @@ public SubmitCromwellRunSetStep(
   @SuppressWarnings(
       "java:S2259") // suppress warning for possible NPE when calling pipelineName.getValue(),
   //  since we do validate that pipelineName is not null in `validateRequiredEntries`
-  public StepResult doStep(FlightContext flightContext)
-      throws InterruptedException, RetryException {
+  public StepResult doStep(FlightContext flightContext) throws InterruptedException {
```

Comment:
curious what is now throwing the `InterruptedException`

Reply:
nothing! removing from this and most doStep and undoStep methods